docs/sql-ref-functions-udf.md

```diff
@@ -1,25 +1,25 @@
 ---
 layout: global
-title: Reference
-displayTitle: Reference
+title: UDFs (User-Defined Functions)
+displayTitle: UDFs (User-Defined Functions)
 license: |
   Licensed to the Apache Software Foundation (ASF) under one or more
   contributor license agreements.  See the NOTICE file distributed with
   this work for additional information regarding copyright ownership.
   The ASF licenses this file to You under the Apache License, Version 2.0
   (the "License"); you may not use this file except in compliance with
   the License.  You may obtain a copy of the License at
- 
+
      http://www.apache.org/licenses/LICENSE-2.0
- 
+
   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
 
-Spark SQL is Apache Spark's module for working with structured data.
-This guide is a reference for Structured Query Language (SQL) for Apache 
-Spark. This document describes the SQL constructs supported by Spark in detail
-along with usage examples when applicable.
+User-Defined Functions (UDFs) are a feature of Spark SQL that allows users to define their own functions when the system's built-in functions are not enough to perform the desired task. To use UDFs in SPARK SQL, users must first define the function, then register the function with SPARK, and finally call the registered function. The User-Defined Functions can act on a single row or act on multiple rows at once.
+
```

Comment:
We don't need to document Hive UDFs here?

Reply:
I will think about this.